etc/profile-m-z/newsboat.profile

```diff
@@ -38,7 +48,7 @@ seccomp
 shell none
 
 disable-mnt
-private-bin gzip,lynx,newsboat,sh
+private-bin gzip,lynx,newsboat,sh,w3m
```

Comment:
w3m seems to require perl when looking at it's profile.

Reply:
normal use is fine, it probably affects advanced features. Given it's an addon here, i don't think that's really a problem.

Reply:
what about this?

`# for advanced features in w3m
#include allow-perl.inc`